iron-flex-layout.html

```diff
@@ -64,9 +64,9 @@
   // removing would be a breaking change, but new projects
   // should never rely on this.
   (function() {
-    var style = document.createElement('style');
-    style.textContent = '[hidden] { display: none !important; }';
-    document.head.appendChild(style);
+  var style = document.createElement('style');
```

Comment:
huh really, no indentation here? interesting.

Reply:
Yeah, it gets rid of indentation for iffes